### DISCUSSION TITLE
How to become a Organization member? that attached many peoples Github profile.

### DISCUSSION THREAD

**Discussion by Suvadip-sana** [recodehive/Stackoverflow-Analysis#402] [General]
How to become a organization member? And how to show the 'Organization' section along with organization name with logo in my profile?

Just like this - 
![image](https://github.com/user-attachments/assets/629f408a-5410-4936-8ea7-3c5c28c442e8)

@sanjay-kv   Please assist me!

**Comment by sanjay-kv** [comment]
https://github.com/Recode-Hive/Support/issues/new?assignees=&labels=invite+me+to+the+community&projects=&template=invitation.yml&title=Please+invite+me+to+the+Recode-Hive+GitHub+Community+Organization

Join here: 
accpet invitation

https://github.com/orgs/recodehive/people
go to your profile and make it public

**Comment by Suvadip-sana** [comment]
@sanjay-kv Thank you sir
